deleteAndLoadSnapsets.sh

```diff
@@ -31,7 +32,24 @@ if [ -z "$COUCHDB_URL" ]; then
 fi
 
 log "Starting"
+log "Clear index: $CLEAR_INDEX"
+log "Static: $STATIC_DATA_DIR"
+log "Build: $BUILD_DATA_DIR"
+log "Node path: $NODE_PATH"
+log "Working directory: `pwd`"
 
+# Set up universal
+git clone https://github.com/GPII/universal.git
+cd universal
+rm -f package-lock.json
```

Comment:
This line is no longer necessary as `package-lock.json` has been removed from the universal repo.

Reply:
Right; however, the subsequent `npm install` commands create a `package-lock.json`.  I don't think it matters since the point is to set up a clone of universal and use it to load and modify the database. That is, I don't think `package-lock.json` affects the database operations.  But, I'm not 100% sure, so, for the time being I've moved the `rm` command to after the `npm install` commands.